src/components/data.js

```diff
@@ -0,0 +1,31 @@
+import {BsPeopleFill} from "react-icons/bs"
+import {BsFillKeyboardFill} from "react-icons/bs"
+import {BsFillFileEarmarkCodeFill} from "react-icons/bs"
+import {BsFillEmojiSmileFill} from "react-icons/bs"
+
+export const FeatureList = [
```

Comment:
(1) This could have been refactored to the util directory
(2) Better naming (e.g. BudCodeFeatures?)